x-pack/packages/kbn-elastic-assistant/impl/knowledge_base/knowledge_base_settings_management/index.tsx

```diff
@@ -412,6 +436,20 @@ export const KnowledgeBaseSettingsManagement: React.FC<Params> = React.memo(({ d
           <p>{i18n.DELETE_ENTRY_CONFIRMATION_CONTENT}</p>
         </EuiConfirmModal>
       )}
+      {duplicateKBItem && (
+        <EuiConfirmModal
+          title={i18n.DUPLICATE_ENTRY_CONFIRMATION_TITLE}
+          onCancel={handleCancelDuplicateEntry}
+          onConfirm={handleDuplicateEntry}
+          cancelButtonText={CANCEL_BUTTON_TEXT}
+          confirmButtonText={i18n.SAVE_BUTTON_TEXT}
+          buttonColor="warning"
```

Comment:
<img width="1081" alt="Screenshot 2024-10-25 at 11 57 12 AM" src="https://github.com/user-attachments/assets/f9ddaafb-a96b-48c3-8087-6b68465d45c9">
I was kind of surprised this was a prop and not something custom you did, the yellow button looks so odd to me.

Reply:
done